src/storage-controller/src/lib.rs

```diff
@@ -1756,34 +1762,13 @@ where
         )
         .await;
     }
-    async fn send_statement_log_updates(
+    async fn record_introspection_updates(
```

Comment:
this seems fine because it takes an `IntrospectionType`, and not a `GlobalId`. However, it should have some documentation, and describe that the `Row`'s MUST have the correct schema, otherwise we will get panics (either on write or read)

Reply:
I extended the comment on the trait method. It's a bit awkward since storage/persist doesn't really care about the schema and doesn't even know what the schema is in some cases. The schemas are defined in the `BuiltinSource` definitions in the catalog, which is an entirely different part of the code.